### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by vikash485**
Solved issue #4

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+- **Copy editing**: fix typos, clarify language, and generally improve the quality of the content.
+- **Formatting**: help keep content easy to read with consistent formatting.
+- **Code**: help maintain and improve the project codebase.
+
+
+### Let's Contribute :+1:
+
+- **Step 1** - Fork this repository.
+- **Step 2** - Clone the repository to your local machine.
+- **Step 3** - Resolve the bugs, mentions provided in the Issues section of the repository. _Also add a description of what changes you have done_.
```
Comment (supminn) [inline CONTRIBUTING.md]: Can you add the steps for local setup in these steps? It is available on the main README.md file

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+- **Step 4** - Add the changes to your repository.
+- **Step 5** - Create a PULL Request. And that's all.
+- **Step 6** - Your PR will be reviewed and merged accordingly, if no changes are required.
```
Comment (supminn) [inline CONTRIBUTING.md]: Add a helper line as follows:
For any queries or suggestions, contact [@supminn](https://twitter.com/supminn) on Direct Message.

**Review by supminn — CHANGES_REQUESTED** [review]
Hi @vikash485 
Thank you for your contribution. I have suggested a few changes. Could you get them incorporated on the same PR.

**Comment by vikash485** [comment]
Sure, will do that now

**Comment by vikash485** [comment]
Please check now @supminn, I have made the changes

**Pull request merged**